src/main/java/seedu/address/model/event/EventName.java

```diff
@@ -4,44 +4,45 @@
 import static seedu.address.commons.util.AppUtil.checkArgument;
 
 /**
- * Represents a Volunteer's name in the volunteer storage.
- * Guarantees: immutable; is valid as declared in {@link #isValidName(String)}
+ * Represents a Event's name in the event storage.
+ * Guarantees: immutable; is valid as declared in {@link #isValidEventName(String)}
  */
 public class EventName {
 
     public static final String MESSAGE_CONSTRAINTS =
             "Names should only contain alphanumeric characters and spaces, and it should not be blank";
 
     /*
-     * The first character of the address must not be a whitespace,
+     * The first character of the name must not be a whitespace,
```

Comment:
can consider changing name to event name?

Reply:
ok i change for u